compiler/vm/vm.nim

```diff
@@ -2834,7 +2833,7 @@ proc rawExecute(c: var TCtx, t: var VmThread, pc: var int): YieldReason =
       else:
         block search:
           for existing in g.cacheSeqs[destKey]:
-            if exprStructuralEquivalent(existing, val, strictSymEquality=true):
+            if exprStructuralEquivalentStrictSymAndComm(existing, val):
```

Comment:
Same comment as above applies. (comments are now considered for trees included in a `CacheTable`)

Reply:
It did consider comment equality before this change, because `exprStructuralEquivalent` prior to this PR checked `nkCommentStmt` nodes for equality.